identity-service/src/remoteConfig.js

```diff
@@ -0,0 +1,21 @@
+const REMOTE_CONFIG_FEATURE = 'remote_config'
+
+// Declaration of remote config variables set in optimizely
+const VARS = {
```

Comment:
probably want to use Object.freeze. Also, maybe call it something like 'REMOTE_VARS'? At the call site it's a bit unclear what 'VARS' refers to